egograph/backend/infrastructure/database/youtube_queries.py

```
@@ -25,23 +25,26 @@ class YouTubeQueryParams:
-def get_watches_parquet_path(bucket: str, events_path: str) -> str:
-    """YouTube視聴履歴のS3パスパターンを生成します。
+def get_watch_events_parquet_path(bucket: str, events_path: str) -> str:
+    """YouTube視聴イベントのS3パスパターンを生成します。
 
     Args:
         bucket: R2バケット名
         events_path: イベントデータのパスプレフィックス
 
     Returns:
-        S3パスパターン（例: s3://egograph/events/youtube/watch_history/**/*.parquet）
+        S3パスパターン（例: s3://egograph/events/youtube/watch_events/**/*.parquet）
```

Ruff (0.15.10)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)